torch/testing/_internal/common_methods_invocations.py

```diff
@@ -293,6 +304,31 @@ def sample_inputs_unary(op_info, device, dtype, requires_grad):
                                     low=low, high=high,
                                     requires_grad=requires_grad)))
 
+
+def sparse_sample_inputs_unary(op_info, device, dtype, requires_grad):
+    """This function generates 2D sparse COO matrices with 20% density. 
+    Generated values are bound to domain if op_info.domain is provided.
+    Generated COO matrices are coalesced and uncoalesced of random shape.
+    """
+    if not op_info.supports_sparse:
+        raise RuntimeError("This op does not support sparse input")
+
+    # generate 2D random coalesced/uncoalesced sparse matrix with density of 20%
+    rh, rw = torch.randint(100, 200, size=(2, )).tolist()
+    low, high = op_info.domain
+    domain = None if (low is None and high is None) else (low + op_info._domain_eps, high - op_info._domain_eps)
```

Comment:
This seems cool. In the future should we consider updating the sparse matrix ctor to be make_sparse_tensor() for consistency with make_tensor(), and look at giving them similar interfaces and logic?